src/lib/components/data-encoder-settings.svelte

```diff
@@ -5,7 +5,7 @@
   import { validateHttpOrHttps, validateHttps } from '$lib/utilities/is-http';
 
   import Modal from './modal.svelte';
-  import Button from './button.svelte';
```

Comment:
It may be worthwhile adding a `$holocene` alias because no one likes relative paths.